src/HDWallet.cpp

```diff
@@ -256,6 +260,10 @@ HDNode getNode(const HDWallet& wallet, TWCurve curve, const DerivationPath& deri
                 // special handling for extended
                 hdnode_private_ckd_cardano(&node, index.derivationIndex());
                 break;
+            case HDWallet::PrivateKeyTypeHD:
+                // special handling for extended
+                hdnode_private_ckd_cardano(&node, index.derivationIndex());
```

Comment:
this can be merged into `PrivateKeyTypeExtended96`

Reply:
Fixed on [https://github.com/trustwallet/wallet-core/pull/1258/commits/f66f4b1458fbe4c6102813cbd717d36b9c287bc3](url)